web/libs/datamanager/src/components/MainView/DataView/Table.scss

```
@@ -0,0 +1,47 @@
+.data-view {
+  flex: 1;
+  display: flex;
+  overflow: hidden;
+  flex-direction: column;
+  background: var(--sand_0);
+}
+
+.no-results {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  &__description {
+    font-size: 16px;
+    text-align: center;
+    color: rgba(#000, 0.6);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
    color: rgba(var(--black), 0.6);
```